src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,30 +1,53 @@
 #!/usr/bin/python
 import sys
 import swsssdk
+import redis
+import argparse
+from multiprocessing import Pool
 
-argc = len(sys.argv)
-if argc == 2 and sys.argv[1] == '-h':
-    print("""
-Example 1: sonic-db-cli CONFIG_DB keys *
-Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
-Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
-Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
-""")
-elif argc < 3:
-    msg = "'Usage: sonic-db-cli <db_name> <cmd> [arg [arg ...]]'. See 'sonic-db-cli -h' for detail examples."
-    print >> sys.stderr, msg
-else:
-    dbname = sys.argv[1]
-    dbconn = swsssdk.SonicV2Connector(use_unix_socket_path=False)
+def ping_single_instance(inst_info):
+    inst_hostname = inst_info['hostname']
+    inst_port = inst_info['port']
+    r = redis.Redis(host=inst_hostname, port=inst_port)
+    rsp = False
+    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname, inst_port)
+    try:
+        rsp = r.ping()
+    except redis.exceptions.ConnectionError as e:
+        pass
+    return 'PONG' if rsp else msg
+
+def ping_all_instances():
+    db_insts = swsssdk.SonicDBConfig.get_instancelist()
```

Comment:
we need get the namesapce info for get_instancelist()

Reply:
shall update, I skipped earlier as these were new functions.